### PR TITLE
[UI] new presale design

### PR DESCRIPTION
**Pull request by peak3d**
- implement i18n locales
- redesign of presale site
- use . as decimal separator

### CODE REVIEW

File: src/components/presale/presale.tsx [hunk 1]
```diff
@@ -277,6 +294,28 @@ class Presale extends Component<PRESALEPROPS, PRESALESTATE> {
     return StoreClasses.store._getPresaleContractAddress() || '';
   }
 
+  _calculateWOLF(val: string | undefined): string {
+    return val && this.state.inputValid
+      ? (parseFloat(val) * 40).toFixed(2).toString()
```
Comment (juztamau5) [inline src/components/presale/presale.tsx:293]: This calculation seems wrong,

wrong-wolf

**Comment by juztamau5** [comment]
@peak3d can you rebase? there's just a renamed file, so you won't have to resolve any conflicts

**Review by juztamau5 — APPROVED** [review]

File: src/components/presale/presale.tsx (hunk 1 above)
Comment (peak3d) [reply src/components/presale/presale.tsx:293]: thx, as discussed: ETH is now darker and better readable, we are currently at 40 WOLF / ETH so calculation is correct.

File: src/components/presale/presale.tsx [hunk 2]
```diff
@@ -286,127 +329,181 @@ class Presale extends Component<PRESALEPROPS, PRESALESTATE> {
...
-      this.state.ethUser,
-      3.0 - this.state.ethInvested
-    );
+    const failureClass = this.state.inputValid ? '' : ' pcr-input-failure';
 
     const { t } = this.props;
 
     return (
-      <div className="tk-aktiv-grotesk-condensed presale-main presale-column">
+      <div className="tk-grotesk-lightbold presale-main presale-column">
```
Comment (juztamau5) [inline src/components/presale/presale.tsx]: classname `presale-column` is unused in the css

Comment (peak3d) [reply src/components/presale/presale.tsx]: thx, fixed

**Pull request merged**